src/script/page/LeftSidebar/panels/Conversations/useSidebarStore.ts

```diff
@@ -60,7 +60,8 @@ const useSidebarStore = create<SidebarStore>()(
       storage: createJSONStorage(() => localStorage),
       partialize: state => ({
         status: state.status,
-        currentTab: state.currentTab === SidebarTabs.PREFERENCES ? SidebarTabs.RECENT : state.currentTab,
+        currentTab:
+          state.currentTab === SidebarTabs.PREFERENCES || SidebarTabs.CONNECT ? SidebarTabs.RECENT : state.currentTab,
```

Comment:
Shouldn't it be more like `[SidebarTabs.PREFERENCES, SidebarTabs.CONNECT].includes(state.currentTab)  ? SidebarTabs.RECENT : state.currentTab`

if state.currentTab is different than preferences, it will resolve to `false` and go to `OR` statement and then `SidebarTabs.CONNECT` itself is always truthy so it will always be `SidebarTabs.RECENT` 🤔